### PR TITLE
[WIP]Specify vuldb path for FreeBSD

### PR DESCRIPTION
**Pull request by justyntemme**
If this Pull Request is work in progress, Add a prefix of “[WIP]” in the title.

# What did you implement:
Added support to define the vulnerability database file manually if previously already downloaded

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 

Fixes # (issue)
https://github.com/future-architect/vuls/issues/473
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:
You don't have to satisfy all of the following.

- [x] Write documentation
- [x] Check that there aren't other open pull requests for the same issue/feature
- [ ] Write tests
- [x] Format your source code by `make fmt`
- [ ] Pass the test by `make test`
- [ ] Provide verification config / commands
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:***  
Yes
# Reference

* https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/

### CODE REVIEW

**Comment by kotakanbe** [comment]
@justyntemme 

Thanks.
Please write how you tested. You can paste the command.

**Comment by justyntemme** [comment]
Currently unable to build on FreeBSD. Do you know the latest branch with a successful build? I was simply using `go get` along with `go build main.go`

Please tell me if this is incorrect and I will get unit tests written when compile works.

**Comment by kotakanbe** [comment]
@justyntemme 

`make build` generates vuls binary.

**Comment by justyntemme** [comment]
You can test by first download a package database with package audit then running
`./vuls scan -config config.toml -package-audit-path /path/to/vuln.db`

This will then scan the servers in config.toml with the database given in the command.

**Comment by justyntemme** [comment]
ping @kotakanbe Please let me know if any further changes are required for this PR

**Comment by kotakanbe** [comment]
Sorry for late reply. I am busy this week. I am going to check the issue. Thanks

File: scan/freebsd.go [hunk 1]
```diff
@@ -176,14 +176,18 @@ func (o *bsd) scanInstalledPackages() (models.Packages, error) {
...
 
 func (o *bsd) scanUnsecurePackages() (models.VulnInfos, error) {
-	const vulndbPath = "/tmp/vuln.db"
-	cmd := "rm -f " + vulndbPath
+
+	vulndbPath := "/tmp/vuln.db"
+	cmd := util.PrependProxyEnv("pkg audit -F -r -f " + vulndbPath)
+	if o.ServerInfo.PkgAuditPath != "" {
+		vulndbPath = o.ServerInfo.PkgAuditPath
+		cmd = util.PrependProxyEnv("pkg audit -f " + vulndbPath)
```
Comment (kotakanbe) [inline scan/freebsd.go]: Why did you remove -r option?
https://www.freebsd.org/cgi/man.cgi?query=pkg-audit&sektion=8&n=1
```
     -r, --recursive
	    Prints packages that depend	on vulnerable packages and are thus
	    potentially	vulnerable as well.
```

Comment (justyntemme) [reply scan/freebsd.go]: My apologies will fix and re submit today.
